### PR TITLE
add questions regarding pull requests

### PR DESCRIPTION
**Pull request by amightyo**
Here are few questions regarding pull requests @ccannon94, @mozzadrella

### CODE REVIEW

File: questions.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+### Pull Requests
+
+* What's the difference between a "pull request" and a "branch"?
```
Comment (mozzadrella) [inline questions.md:3]: Hello @amightyo! 
A pull request is a collaboration feature where you are asking for review from members of your team on one branch (or several!). Pull requests are a GitHub feature.

A branch is a pointer to a commit, and a Git feature.

File: questions.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+* How do we handle conflict during a pull request?
```
Comment (mozzadrella) [inline questions.md:4]: So glad you asked! Here is a tutorial on merge conflicts in the classroom: https://github.blog/2018-08-22-merge-conflicts-in-the-classroom/

**Review by mozzadrella — APPROVED** [review]
Hope this helps--let me know if you have additional questions :)

File: questions.md (hunk 2 above)
Comment (amightyo) [reply questions.md:4]: Thank you! 💯

File: questions.md (hunk 1 above)
Comment (amightyo) [reply questions.md:3]: Thank you! 👍

**Pull request merged**